src/documentation/verifier.cpp

```diff
@@ -271,6 +273,57 @@ std::string DirectoryVerifier::type() const {
     return "Directory";
 }
 
+TestResult DateTimeVerifier::operator()(const ghoul::Dictionary& dict,
+    const std::string& key) const
+{
+    TestResult res = StringVerifier::operator()(dict, key);
+    if (!res.success) {
+        return res;
+    }
+
+    std::string dateTime = dict.value<std::string>(key);
+    std::string format = "%Y %m %d %H:%M:%S"; // YYYY MM DD hh:mm:ss
+
+    std::tm t = {};
+    std::istringstream ss(dateTime);
+    ss >> std::get_time(&t, format.c_str());
+
+    // first check format (automatically checks if valid time)
+    if (ss.fail()) {
+        res.success = false;
+        TestResult::Offense off;
+        off.offender = key;
+        off.reason = TestResult::Offense::Reason::Verification;
+        off.explanation = "Not a valid format, should be: YYYY MM DD hh:mm:ss";
+        res.offenses.push_back(off);
+    }
+    // then check if valid date
+    else {
+        // normalize e.g. 29/02/2013 would become 01/03/2013
+        std::tm t_copy(t);
+        time_t when = mktime(&t_copy);
+        std::tm *norm = localtime(&when);
```

Comment:
We attach the pointer to the type as it is a clarifier of the type and not the name